nb/projects/plotter/mount.nb

```diff
@@ -79,6 +83,8 @@ const toothedReel = Group(
   .as('toothedReel')
   .view(rx(1 / 2));
 
+/*
+// FIX: These are too slow.
```

Comment:
We should use the wasm thread support to compute our threads faster 😛

Reply:
Yeah, although I think this is a different problem.

I'll need to come back and figure out what's happening here.

Reply:
Yeah, it wasn't a particularly insightful comment. I just wanted to make the joke 😁